bpf/bpf_host.c

```diff
@@ -441,6 +441,8 @@ handle_ipv4(struct __ctx_buff *ctx, __u32 secctx,
 	struct endpoint_info *ep;
 	void *data, *data_end;
 	struct iphdr *ip4;
+	bool l2_hdr_required = true;
+	int l3_off = ETH_HLEN;
```

Comment:
We can reduce the scope here, and just define these variables inside the relevant section:

```
ep = lookup_ip4_endpoint(ip4);
if (ep) {
	bool l2_hdr_required = true;
	int l3_off = ETH_HLEN;

	...
}
```

Reply:
Compiler will complain "error: mixing declarations and code is a C99 extension [-Werror,-Wdeclaration-after-statement]" - -